src/decode.rs

```diff
@@ -3992,7 +3992,7 @@ pub(crate) unsafe fn rav1d_decode_tile_sbrow(
                 let x1 = (4 * (t.b.x + sb_step) * d >> ss_hor) + rnd >> shift;
 
                 for x in x0..cmp::min(x1, n_units) {
-                    let px_x = x << unit_size_log2 + ss_hor;
+                    let px_x = x << unit_size_log2 + ss_hor as u8;
```

Comment:
```suggestion
                    let px_x = x << unit_size_log2 + ss_hor;
```

I think this is redundant since you already made `ss_hor` a `u8` above.

Reply:
Must be a different `ss_hor`. The one here is a `c_int`.